src/Faker/Provider/ne_NP/Payment.php

```diff
@@ -0,0 +1,111 @@
+
+    /**
+     * List of digital wallets sorted in alphabetical order.
+     *
+     * @var string[]
+     */
+    protected static $digitalWallets = ['CellPay', 'CG Pay', 'Chito Paisa', 'DigiPay', 'dPaisa', 'EnetPay', 'eSewa', 'Fonepay', 'GME Pay', 'iCash', 'IME Pay', 'Ipay', 'Khalti', 'Kurakani Pay', 'Lenden', 'Mobalet', 'MOCO', 'Mohar', 'Moru', 'N-Cash', 'Namaste Pay', 'PayTime', 'PayWell', 'PrabhuPAY', 'QPay', 'SajiloPay', 'WePay'];
+
+    /**
+     * List of Swift Codes in alphabetical order.
+     *
+     * @see https://www.theswiftcodes.com/nepal/
+     *
+     * @var string[]
+     */
+    protected static $swiftCodes = ['ADBLNPKA', 'BOKLNPKA', 'CCBNNPKA', 'CIVLNPKA', 'CTZNNPKA', 'EVBLNPKA', 'GLBBNPKA', 'HIMANPKA', 'KMBLNPKA', 'LXBLNPKA', 'MBLNNPKA', 'MBNLNPKA', 'NARBNPKA', 'NBOCNPKA', 'NBOCNPKANRD', 'NEBLNPKA', 'NIBLNPKT', 'NICENPKA', 'NMBBNPKA', 'NPBBNPKA', 'NRBLNPKA', 'NRBLNPKAFED', 'NSBINPKA', 'NSBINPKA001', 'PCBLNPKA', 'PRVUNPKA', 'RBBANPKA', 'SCBLNPKA', 'SIDDNPKA', 'SNMANPKA', 'SRBLNPKA'];
+
+    /**
+     * @example 'Agricultural Development Bank'
+     */
+    public function commercialBank(): string
+    {
+        return static::randomElement(static::$commercialBanks);
+    }
+
+    /**
+     * @example 'Nepal Infrastructure Bank'
+     */
+    public function developmentBank(): string
+    {
+        return static::randomElement(static::$developmentBanks);
+    }
+
+    /**
+     * @example 'Gurkhas Finance'
+     */
+    public function financeCompany(): string
+    {
+        return static::randomElement(static::$financeCompanies);
+    }
+
+    /**
+     * @example 'Adarsha Laghubitta Bittiya Sanstha'
+     */
+    public function microFinance(): string
+    {
+        $suffix = ' Laghubitta Bittiya Sanstha';
+
+        return static::randomElement(static::$microFinances) . $suffix;
+    }
+
+    /**
+     * @example 'Khalti'
+     */
+    public function digitalWallet(): string
+    {
+        return static::randomElement(static::$digitalWallets);
+    }
+
+    /**
+     * @example 'ADBLNPKA'
+     */
+    public function swiftCode(): string
+    {
+        return static::randomElement(static::$swiftCodes);
+    }
+
+    /**
+     * @example '00454689832792' or 'S49646367883667'
+     */
+    public function bankAccountNumber(): string
+    {
+        $prefixType = self::randomElement(['alpha', 'zero']);
+        $format = $prefixType === 'alpha' ? '[A-Z][1-9]{8,19}' : '[0]{2}[1-9]{7,18}';
```

Comment:
I think its pretty rough to do a string compare for a 50% compare

Reply:
I changed to boolean.